.eslintignore

```diff
@@ -3,3 +3,5 @@ coverage
 dist
 patches
 yarn.lock
+test
```

Comment:
what are the lint failures? If you think some are wrong, instead of disabling lint we should tailor the rules to the ones that should be followed.

Reply:
Thanks for catching that. I had linting issues in `support.js` because it wasn't in the `test/e2e` directory, where the synpress-specific eslint file is located.